File: AD-HOC/1192/main.py
n = int(input())
for _ in range(n):
    seq = input()
    digit1, letter, digit2 = int(seq[0]), seq[1], int(seq[2])
    if (digit2 == digit1):
        print(digit2 * digit1)
    elif (letter.isupper()):
        print(digit2 - digit1)
    else:
        print(digit2 + digit1)

File: AD-HOC/1192/main.c
#include <ctype.h>
#include <stdio.h>

int main() {
    int n, digit1, digit2;
    char letter;
    scanf("%d", &n);
    for (int i = 0; i < n; i++) {
        scanf("%d%c%d", &digit1, &letter, &digit2);
        if (digit2 == digit1)
            printf("%d\n", digit2 * digit1);
        else if (isupper(letter))
            printf("%d\n", digit2 - digit1);
        else
            printf("%d\n", digit2 + digit1);
    }
    return 0;
}
